### ISSUE TITLE
exec with an alias fails

### ISSUE THREAD

**Issue by dragonbitestail** [gugod/App-perlbrew#725]
Per 
https://perlbrew.pl/Perlbrew-In-Shell-Scripts.html

When using exec with an alias, the Perl script does not seem to get executed. No output:

    /opt/perl5/bin/perlbrew exec -q --with perl-latest perl /ops/app/plcron/bin/plcron.pl

However same works when using the Perl version the alias points to.  My script currently just
exits due to missing dependencies:

```
/opt/perl5/bin/perlbrew exec -q --with perl-5.35.0 perl /ops/app/plcron/bin/plcron.pl
Can't locate YAML/Any.pm in @INC (you may need to install the YAML::Any module) (@INC contains: /opt/uad/lib/perl /opt/uad/lib/perl/lib/perl5 /opt/perl5/perls/perl-5.35.0/lib/site_perl/5.35.0/x86_64-linux /opt/perl5/perls/perl-5.35.0/lib/site_perl/5.35.0 /opt/perl5
/perls/perl-5.35.0/lib/5.35.0/x86_64-linux /opt/perl5/perls/perl-5.35.0/lib/5.35.0) at /ops/app/plcron/bin/plcron.pl line 17.
BEGIN failed--compilation aborted at /ops/app/plcron/bin/plcron.pl line 17.
```

I can "use" the same Perl via the alias without issue. Just "exec" seems confused:

```
bash-4.2$ perlbrew use perl-latest

bash-4.2$ which perl && perl -v | head -2

/opt/perl5/perls/perl-latest/bin/perl

This is perl 5, version 35, subversion 0 (v5.35.0) built for x86_64-linux
```

Maybe the perlbrew exec command is doing a plain file check for the Perl version and need to also check for sym. link?

EDIT:

This is Perlbrew 0.92

See line 2770 in  sub "run_command_exec"

```
"next if -l $self->root->perls ($i->{name}); # Skip Aliases"
```

Maybe a good reason for this but, it escapes me.

**Comment by gugod** [comment]
Ignoring alias in `exec` was for preventing duplicate runs -- see this commit 61ae08cd09fc452043218e9803b4195f8801cdf5 in early history.

I guess we should make it not skipping alias when aliases are explicitly specified.

**Comment by gugod** [comment]
> I guess we should make it not skipping alias when aliases are explicitly specified.

This is implemented in #731

**Comment by dragonbitestail** [comment]
Appreciate you addressing this.

  I see I was trying to use this Perlbrew option unnecessarily.

  At first I was deploying Perlbrew target hosts and was trying to use this option to run scripts on those hosts.
  This was a wrong-headed approach.

  Not long after reporting I started using Perlbrew to build a version of Perl with additional modules installed and package the whole bin. tree into a tar.gz on a build/staging host.
 Part of the build process creates a sym. link to the Perl built under Perlbrew that I wanted as "latest"
  
  Any user which uses that built version inserts the sym. link path into env. PATH
  Then script bang line:
  #!/bin/env perl
